### DISCUSSION TITLE
Feature: Allow Overrides of Labels/Arguments for Post Type/Taxonomies

### DISCUSSION THREAD

**Discussion by jimtrue** [pods-framework/pods#6684] [Ideas]
## Issue Overview
This is a holding issue for a Feature/Enhancement to allow overrides of the labels of WordPress default elements (Post Title, Editor, Author, Featured Image, Taxonomies) as well as placeholders for those and additional `add-post-type-support` options for extending Pages/Posts, or other plugins.

A lot of this goes hand-in-hand with #2760 and #3469 and any way we can make our 'Pods' Admin UI for a Content Type be more 'inclusive' and helpful. These are Quality of Life Improvements. Possibly pull together any of these that are considered Admin UI overhaul items.

Task list for Jim 

- [ ] Find all Admin UI type Feature Enhancements that are feasible
- [ ] Find any Quality of Life that currently require code additions to handle that should be built-into our 'extend' existing framework. (ie pre_get_post filters for Taxonomies, Authors, etc., any add-post-type-support code editions). Some of these may go hand-in-hand with Gutenberg handling. 
- [ ] Address any Advanced Options in Pods Admin that should be migrated into the Add New Wizard, ie Archives, Favicon, Associated taxonomies.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## WordPress Environment

<details>
```
Copy and paste your System Details from **Pods Admin > Settings > Debug Information** in WordPress admin here.
```
</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

**Comment by jimtrue** [comment]
## Options to add to Add New Wizard
- Allow option to Enable Archives as part of the Add New Wizard
- Set Archive Slug (if enabled Archives)
- Do you have any taxonomies you'd like to associate? (unless this is moved into a better handling for Taxonomy management)

**Comment by jimtrue** [comment]
Post Title Filter for Placeholder is `'enter_title_here'`; the idea would be for us to be able to provide a 'place' for this in the Admin UI tab for the Post.

**Comment by AstroJetson** [comment]
I'd vote for this "Allow option to Enable Archives as part of the Add New Wizard"

I also like the way to set the prompt for the the title for the POD, I just did it using the description field.
